composer.json

```diff
@@ -135,8 +135,7 @@
             "Symfony\\Component\\": "src/Symfony/Component/"
         },
         "classmap": [
-            "src/Symfony/Component/Intl/Resources/stubs",
-            "src/Symfony/Bridge/ProxyManager/Legacy/ProxiedMethodReturnExpression.php"
```

Comment:
this should be kept actually
sorry for the misleading comment, the line I had in mind has been removed earlier

Reply:
reverted